### DISCUSSION TITLE
failed to fetch https://docker.artifactory.magmacore.org/artifactory/helm/index.yaml : 403 Forbidden

### DISCUSSION THREAD

**Discussion by ovasens** [magma/magma#10715] [General]
i'am trying to install orchestrator but i have issue below

**terraform apply** 
`
module.orc8r.aws_elasticsearch_domain.es[0]: Modifications complete after 4s [id=arn:aws:es:us-west-2:977828861673:domain/orc8r-es]
module.orc8r.data.aws_iam_policy_document.es-management[0]: Reading... [id=3190709067]
module.orc8r.data.aws_iam_policy_document.es-management[0]: Read complete after 0s [id=3190709067]
╷
│ Error: ooks like "https://docker.artifactory.magmacore.org/artifactory/helm" is not a valid chart repository or cannot be reached: failed to fetch https://docker.artifactory.magmacore.org/artifactory/helm/index.yaml : 403 Forbidden
│
│   with module.orc8r-app.helm_release.orc8r,
│   on .terraform/modules/orc8r-app/orc8r/cloud/deploy/terraform/orc8r-helm-aws/main.tf line 27, in resource "helm_release" "orc8r":
│   27: resource "helm_release" "orc8r" {
│
`
some one can help me please ?

**helm repo list**
NAME            URL
stable          https://charts.helm.sh/stable
magmacore       https://docker.artifactory.magmacore.org/artifactory/helm
incubator       https://charts.helm.sh/incubator

**Comment by ShubhamTatvamasi** [comment]
You have to add helm repo like this:
```bash
helm repo add orc8r https://artifactory.magmacore.org/artifactory/helm/
helm repo update
```

**Reply by ovasens** [reply]
@ShubhamTatvamasi I did it but i have the same issue

 **helm repo list**
NAME            URL
stable          https://charts.helm.sh/stable
magmacore       https://docker.artifactory.magmacore.org/artifactory/helm
incubator       https://charts.helm.sh/incubator
helm_release    https://docker.artifactory.magmacore.org/artifactory/helm
magma           https://docker.artifactory.magmacore.org/artifactory/helm
orc8r           https://artifactory.magmacore.org/artifactory/helm/

**I did terraform init && terraform apply  but i have the same error** 
`module.orc8r.data.aws_iam_policy_document.es-management[0]: Read complete after 0s [id=3190709067]
╷
│ Error: looks like "https://docker.artifactory.magmacore.org/artifactory/helm" is not a valid chart repository or cannot be reached: failed to fetch https://docker.artifactory.magmacore.org/artifactory/helm/index.yaml : 403 Forbidden
│
│   with module.orc8r-app.helm_release.orc8r,
│   on .terraform/modules/orc8r-app/orc8r/cloud/deploy/terraform/orc8r-helm-aws/main.tf line 27, in resource "helm_release" "orc8r":
│   27: resource "helm_release" "orc8r" {
│
╵
`

**Reply by ovasens** [reply]
**helm search repo orc8r**
NAME                            CHART VERSION   APP VERSION     DESCRIPTION
helm_release/cwf-orc8r          v1.6            1.0             A Helm chart for magma orchestrator's cwf module
helm_release/fbinternal-orc8r   v1.6            1.0             A Helm chart for magma orchestrator's fbinterna...
helm_release/feg-orc8r          v1.6            1.0             A Helm chart for magma orchestrator's feg module
helm_release/lte-orc8r          v1.6            1.0             A Helm chart for magma orchestrator's lte module
helm_release/orc8r              v1.6            1.0             A Helm chart for magma orchestrator
helm_release/wifi-orc8r         v1.6            1.0             A Helm chart for magma orchestrator's wifi module
magma/cwf-orc8r                 v1.6            1.0             A Helm chart for magma orchestrator's cwf module
magma/fbinternal-orc8r          v1.6            1.0             A Helm chart for magma orchestrator's fbinterna...
magma/feg-orc8r                 v1.6            1.0             A Helm chart for magma orchestrator's feg module
magma/lte-orc8r                 v1.6            1.0             A Helm chart for magma orchestrator's lte module
magma/orc8r                     v1.6            1.0             A Helm chart for magma orchestrator
magma/wifi-orc8r                v1.6            1.0             A Helm chart for magma orchestrator's wifi module
magmacore/cwf-orc8r             v1.6            1.0             A Helm chart for magma orchestrator's cwf module
magmacore/fbinternal-orc8r      v1.6            1.0             A Helm chart for magma orchestrator's fbinterna...
magmacore/feg-orc8r             v1.6            1.0             A Helm chart for magma orchestrator's feg module
magmacore/lte-orc8r             v1.6            1.0             A Helm chart for magma orchestrator's lte module
magmacore/orc8r                 v1.6            1.0             A Helm chart for magma orchestrator
magmacore/wifi-orc8r            v1.6            1.0             A Helm chart for magma orchestrator's wifi module
orc8r/cwf-orc8r                 v1.6            1.0             A Helm chart for magma orchestrator's cwf module
orc8r/fbinternal-orc8r          v1.6            1.0             A Helm chart for magma orchestrator's fbinterna...
orc8r/feg-orc8r                 v1.6            1.0             A Helm chart for magma orchestrator's feg module
orc8r/lte-orc8r                 v1.6            1.0             A Helm chart for magma orchestrator's lte module
orc8r/orc8r                     v1.6            1.0             A Helm chart for magma orchestrator
orc8r/wifi-orc8r                v1.6            1.0             A Helm chart for magma orchestrator's wifi module

**Reply by ShubhamTatvamasi** [reply]
This is very unlikely. Can you ping me on slack so we can setup a call and resolve this issue?
https://slack.magmacore.org/

**Reply by ovasens** [reply]
to solve the issue, on main.tf  param below must be left blank 
  helm_user       = ""
  helm_pass       = ""
 docker_user     = ""
  docker_pass     = ""

thanks @ShubhamTatvamasi
